base/parallel/Parallel.jl

```diff
@@ -0,0 +1,76 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+module Parallel
+
+# imports for extension
+import Base: getindex, wait, put!, take!, fetch, isready, push!, length,
+             hash, ==, connect, kill, serialize, deserialize, close
+
+# imports for use
+import Base: Process, Semaphore, JLOptions, AnyDict, buffer_writes, wait_connected,
+             VERSION_STRING, sync_begin, sync_add, sync_end, async_run_thunk,
+            binding_module, notify_error
+
+# The following are required but cannot be imported at this
+# time as they are included after Parallel in sysimg.jl
```

Comment:
I remember trying to do so and ran into some cross dependencies the last time around. Will check again if I can prune this list somewhat, else I'll just let it be for now.